README.md

```diff
@@ -84,6 +84,14 @@ This is a function that, for given source, returns a derivation with a Composer
 
 - Either `lockFile` containing an explicit path to `composer.lock` file, or `src`, which is the source directory/derivation containing the file.
 
+### `c4.fetchComposerDepsImpure`
```

Comment:
Maybe we should just re-use `c4.fetchComposerDeps` and change the implementation when passed `__impure` argument. We could even pass `hash` to get FOD.